csharp/ColumnEncryptionPropertiesBuilder.cs

```diff
@@ -37,20 +51,34 @@ public ColumnEncryptionPropertiesBuilder Key(byte[] key)
             return this;
         }
 
+        /// <summary>
+        /// Set the metadata associated with the encryption key for the column.
+        /// </summary>
+        /// <param name="keyMetadata">A string containing the metadata associated with the encryption key.</param>
+        /// <returns>This builder instance.</returns>
         public ColumnEncryptionPropertiesBuilder KeyMetadata(string keyMetadata)
         {
             ExceptionInfo.Check(ColumnEncryptionPropertiesBuilder_Key_Metadata(_handle.IntPtr, keyMetadata));
             GC.KeepAlive(_handle);
             return this;
         }
 
+        /// <summary>
+        /// Set the key ID associated with the column's encryption key.
+        /// </summary>
+        /// <param name="keyId">A unique identifier for the encryption key.</param>
```

Comment:
```suggestion
        /// <param name="keyId">An identifier for the encryption key.</param>
```
I'm not sure that the unique qualifier makes sense here. This can be any string that readers can use to identify the required key when they decrypt the file (eg. when implementing a class that inherits from `DecryptionKeyRetriever`), so I guess keys need to be uniquely identifiable from the set of known possible keys. But there's no real constraint on what this can be as far as ParquetSharp and Arrow C++ are concerned.

This serves the same purpose as `KeyMetadata` but with slightly different semantics. The C++ library allows `KeyMetadata` to be arbitrary bytes rather than needing it to be a valid string, but in ParquetSharp they're both strings so there isn't really any difference here.

Reply:
Interesting, thanks for the insight!